pubnub-memberships/src/main/kotlin/com/pubnub/entities/MembershipExtension.kt

```diff
@@ -243,6 +270,12 @@ fun PubNub.removeMemberships(
     ) { pnMemberArrayResult -> pnMemberArrayResult.toSpaceMembershipResult() }
 }
 
+fun PubNub.removeMemberships(
+    spaceId: SpaceId,
+    firstUserId: UserId,
+    vararg userIds: IUserId
```

Comment:
I had to introduce an interface here to avoid the problem of value classes not yet being supported by varargs as stated here https://github.com/Kotlin/KEEP/issues/104#issuecomment-757998733. When it's no longer an issue we can remove it. I would appreciate some help with naming that though :)